README.md

```diff
@@ -0,0 +1,158 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>Javascript</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://github.com/">Github</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add Books to a book list**
+- **Delete Books from the book list**
+- **Save data to local storage**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+#### <a href="https://kamranahmad123.github.io/ES6-booksList/?">Live Demo Link</a>
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+
+```sh
+  git clone https://github.com/kamranahmad123/ES6-booksList
+```
+
+
+
+### Deployment
+
+You can deploy this project using 
+> [Github Pages](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site)
+
+> [github html preview](https://github.com/kamranahmad123/ES6-booksList) and paste index adress into it to view.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors 
+<a name="authors"></a>
+
+👤 **Kamran Ahmad**
+__Kamran Ahmad__
+- GitHub: [@githubhandle](https://github.com/kamranahmad123) 
+- Twitter: [@twitterhandle](https://twitter.com/handel)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kamran-khan-9a293b113/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Java script only, to do a simple CRUD**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give me a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+ We would like to thank all our fellow micronauts.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+- **What is this project?**
+
+  - It's a  project of micoverse curriculum entitled awesome books.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/kamranahmad123/ES6-booksList/blob/modules/LICENSE) licensed.
```

Comment:
- [ ] Great job. Your MIT link is broken please kindly fix it. we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.